guiclient/enterPoReceipt.cpp

```
@@ -366,8 +366,11 @@ void enterPoReceipt::sPost()
          return;
         }
       }
-      else if (tohead.lastError().type() != QSqlError::NoError)
+      else
```

Leave as else if. We only want to throw an error if the function threw an error, if it found no valid records, that just means the source itemlocdist doesn't need to be created (for example, it isn't controlled).